src/Cluster/Health.php

```diff
@@ -11,6 +11,26 @@
  * @author Ray Ward <ray.ward@bigcommerce.com>
  *
  * @see https://www.elastic.co/guide/en/elasticsearch/reference/current/cluster-health.html
+ *
+ * @phpstan-type HealthData = array{
```

Comment:
See my comment in the other PR related to these. The question is likely mostly for my understanding.